tex/latex/fks/problems.tex

```diff
@@ -95,7 +96,10 @@
 průměr~\@probavg;
 \plural{\@probsolvers}{řešil}{řešili}{řešilo}{}~\@probsolvers{}~\plural{\@probsolvers}{student}{studenti}{studentů}}}
```

Comment:
Nemělo by to být taky přeloženo? (Když už je přeloženo bodování? Případně nějaký vysvětlující comment.)